docs/semgrep-ci/configuration-reference.md

```diff
@@ -22,20 +22,27 @@ SEMGREP_RULES="p/security-audit p/secrets"
 
 ## Diff-aware scanning (`SEMGREP_BASELINE_REF`)
 
-For [diff-aware scans](overview.md#features), set this variable
-to the git ref (branch name, tag, or commit hash) to use as a baseline.
-For example, to report findings newly added
-since branching off from your `main` branch, set
+For [diff-aware scans](overview.md#features), set this variable to create a baseline. This option limits scan results to those introduced after the git commit, in a branch, or tag. For example, you have a repository with 10 commits. You set the commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10.
```

Comment:
> set this variable to create a baseline

This sounds like a "baseline" is a thing that will be created and kept persistently. Maybe saved on the semgrep app server, or something.

How about: "set this variable to compare to a baseline"

Reply:
> You set the commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10.

How about:

You set the 8th commit as the baseline. Consequently, Semgrep only returns scan results introduced by changes in the 9th and 10th commits.

Reply:
@underyx good idea. Changing to: `set this variable to compare to a baseline.`